src/orders/dto/create-order.dto.ts

```diff
@@ -13,11 +13,15 @@ export class CreateOrderDto {
 
   @IsNotEmpty()
   @IsObject()
-  pickupLocation: { latitude: number; longitude: number };
+  pickupLocation: {
+    coordinates: [number, number];
+  };
 
   @IsNotEmpty()
   @IsObject()
-  dropOffLocation: { latitude: number; longitude: number };
+  dropOffLocation: {
+    coordinates: [number, number];
+  };
 
```

Comment:
You can create an array of locations. Because you will able to add more than one location